Doc/library/decimal.rst

```diff
@@ -2115,3 +2115,17 @@ Alternatively, inputs can be rounded upon creation using the
 
    >>> Context(prec=5, rounding=ROUND_DOWN).create_decimal('1.2345678')
    Decimal('1.2345')
+
```

Comment:
The speedups are in `libmpdec`. Recent versions of `pypy3` also use `libmpdec`, so they have the feature, too (I checked `PyPy 5.8.0-beta0`).

So I'd mention `libmpdec`, `CPython` and `PyPy3`.

In order to give people a more accurate idea of the algorithms used, I'd mention that `libmpdec` uses `Karatsuba multiplication` for medium sized numbers and the `Number Theoretic Transform` for very large numbers.

Reply:
Thanks @skrah.  I've included your recommendations in the latest commit.